src/schemas/json/clangd.json

```diff
@@ -607,6 +615,10 @@
         "DeducedTypes": {
           "description": "Whether to enable inlay-hints for deduced types\nhttps://clangd.llvm.org/config.html#deducedtypes",
           "type": "boolean"
+        },
+        "Designators": {
+          "description": "A boolean that enables/disables inlay-hints for designators in aggregate initialization\nhttps://clangd.llvm.org/config.html#designators",
```

Comment:
Required fix: rewrite description to begin it with `Whether to`, consistency violation.

Reply:
Link to docs doesn't work. 🤔